multiqc/modules/sambamba/markdup.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from collections import OrderedDict
+import os
+import logging
+import re
+
+from multiqc import config
+from multiqc.plots import bargraph
+
+log = logging.getLogger(__name__)
+
+class SambambaMarkdupMixin:
+
+    """ Find and parse Sambamba Markdup output log files and calculate duplication rate """
+
+    def parse_sambamba_markdup(self):
+
+        # Clean sample name from 'markdup_sample_1' to 'sample_1'
+        # Find and load sambamba logs to markdup_data_dict.
+        # Regex for key phrases and calculate duplicate rate.
+        # Give user warning if redundant samples are found.
+
+        self.markdup_data_dict = dict()
+
+        for f in self.find_log_files("sambamba/markdup"):
+
+            sample_name = self.markdup_clean_s_name(f["s_name"])

```

Comment:
```suggestion
            sample_name = self.clean_s_name(f["s_name"].replace("markdup_", ""), f['root'])
```

See comment below for discussion..

Reply:
This worked great.